default/stringtables/en.txt

```diff
@@ -7917,6 +7920,11 @@ invalid ship part class
 PC_DIRECT_WEAPON
 Direct Weapon
 
+PC_DIRECT_WEAPON_DESC
+'''Direct-fire weapons acquire targets without third-party assistance (a friendly ship or planet).
```

Comment:
There are no indirect fire weapons, so you don't need to make a distinction about it in the description. Perhaps "Direct-fire weapons attack and can damage targets each combat round (unlike fighters, which must first be launched before they can attack)."